src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/SlhDsa/SlhDsaImplementationTests.cs

```diff
@@ -83,6 +394,105 @@ public void NistKeyGenerationTest(SlhDsaTestData.SlhDsaKeyGenTestVector vector)
             }
         }
 
+        #endregion NIST test vectors
+
+        [Theory]
+        [MemberData(nameof(SlhDsaTestData.AlgorithmsData), MemberType = typeof(SlhDsaTestData))]
+        public void ImportPkcs8WithTrailingData(SlhDsaAlgorithm algorithm)
```

Comment:
This looks redundant to ArgumentValidation_MalformedAsnEncoding

Reply:
The other one also tests the case when SlhDsa.IsSupported is false while this one depends on SlhDsa support. This one uses a valid input as the base to isolate the failure case more so I think they're both useful.